app/models/post.rb

```diff
@@ -6,7 +6,8 @@ class Post < ApplicationRecord
   validates :title, presence: { message: " can't be blank" }
   validate :acceptable_image
   validates_presence_of :account_id, :community_id
-  
+  validate :at_least_one_present
```

Comment:
"at least one present" seems to be very generic, can you change this method to be more specific? what are you checking for? this is not clear

Reply:
Method name fixed now